tests/test_comp_with_appdirs.py

```diff
@@ -62,6 +62,12 @@ def test_compatibility(params: dict[str, Any], func: str) -> None:
         }
         if func in msg:  # pragma: no cover
             pytest.skip(f"`appdirs.{func}` {msg[func]} on macOS")  # pragma: no cover
+    if sys.platform == "linux":
```

Comment:
Can you add a changelog entry here https://github.com/platformdirs/platformdirs/blob/main/CHANGES.rst to release this?